.github/workflows/codeql.yml

```diff
@@ -2,8 +2,16 @@ name: "CodeQL"
 
 on:
   push:
+    paths-ignore:
+      - '**/*.md'
+      - 'dev/**'
+      - 'helm/**'
     branches: [ 'master', '0.6.x', '0.7.x', '0.7.1.x', '0.7.2.x', '0.8.0', '0.8.1', '0.8.2', '0.8.3', '0.9.0', '0.9.1' ]
   pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - 'dev/**'
+      - 'helm/**'
```

Comment:
```suggestion
      - '**/*.md'
      - 'dev/**'
      - 'docs/**'
      - 'helm/**'
```

Reply:
@abhishekrb19 According to the discussion above, we can include the docs dir here, thanks :smile: